components/esp_lvgl_port/Kconfig

```diff
@@ -0,0 +1,14 @@
+menu "ESP LVGL Portation"
+    config LVGL_PORT_TASK_PRIORITY
+        int "LVGL task priority"
+        default 4
+        help
+            The component will create a task that will periodically handle LVGL operation in lv_timer_handler().
+
+    config LVGL_PORT_TICK_PERIOD_MS
+    int "LVGL tick period [ms]"
+    default 5
+    range 1 100
+    help
+        Period of LVGL tick timer.
```

Comment:
We don't have to do it now; just an idea for future improvement:

We should once-and-for-all find a way how to get time to LVGL without periodic interrupts. e.g. with `esp_timer_get_time`